monitoring_versioned_docs/version-22.10/i-vertix-vault/vault.md

```diff
@@ -0,0 +1,26 @@
+---
+id: vault
+title: Vault
+---
+
+## VAULT
+
+Vault is a function of i-Vertix IT Monitoring that acts as an internal password safe.
+
+
+This function amalgamates credentials that will be used for the following features:
```

Comment:
non si capisce, scrivere meglio

esempio: The i-Vertix Vault will be used by the following features:

Reply:
Volevo far capire che questa funzione centralizza le credenziali da usare su le seguenti funzioni.
Quello che scrivi mi va bene.

Reply:
Fatto.